### DISCUSSION TITLE
Inner connection of a iRangeX plus

### DISCUSSION THREAD

**Discussion by AlbertFarguell** [pascallanger/DIY-Multiprotocol-TX-Module#908] [Q&A]
I just bought an irangeX plus 4-in-1 and I want to use it without its case in my radio. Does anyone know how to connect it to work in serial mode? The radio works perfectly with an atmega based module, but I want to upload more protocols in it. Thank you in advance.

**Comment by pascallanger** [comment]
Apart from heartbeat they all need to be connected:
![Module bay pin out](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/assets/6367598/dccbef03-ee32-4eef-9063-9673f80ff750)
From a radio perspective CPPM is TX, SPORT is RX, VMain and GND are power supply

**Reply by AlbertFarguell** [reply]
Thank you Pascal, very kind of you.
